game/core/src/me/lihq/game/people/NPC.java

```diff
@@ -192,6 +192,16 @@ public NPC setMotive(String motive)
     }
 
     /**
+     * This method sets the NPCs moveability to the value of the parameter
+     *
+     * @param canMove - whether or not to set allow the NPC to move
+     */
+    public void setCanMove(boolean canMove)
```

Comment:
We get to control what the value is set to. Say we had an integer, we can bound it using a setter.
Look at NPC.setKiller(true). That is securing that it only makes them the killer if they arent the victim. If it was public, someone could just do .isKiller = true and then the killer could also be the victim